data/frame_meta.go

```diff
@@ -13,16 +13,22 @@ type FrameMeta struct {
 	// Datasource specific values
 	Custom interface{} `json:"custom,omitempty"`
 
-	// Stats is TODO
-	Stats interface{} `json:"stats,omitempty"`
-
-	// This is the raw query sent to the underlying system.  All macros and templating
-	// as been applied.  When metadata contains this value, it will be shown in the query inspector
-	ExecutedQueryString string `json:"executedQueryString,omitempty"`
+	// Query Stats
+	Stats []QueryResultMetaStat `json:"stats,omitempty"`
 
 	// Notices provide additional information about the data in the Frame that
 	// Grafana can display to the user in the user interface.
 	Notices []Notice `json:"notices,omitempty"`
+
+	// Used to track transformation IDs that were part of the processing
+	Transformations []string `json:"transformations,omitempty"`
+
+	// Currently used to show results in Explore only in preferred visualisation option
+	PreferredVisualisationType string `json:"preferredVisualisationType,omitempty"`
```

Comment:
Would change to `PreferredVisualization` (spelling change - https://ell.stackexchange.com/a/5745) and drop "Type" from the name.

The type `string` should be VisualizationType, and we can define the valid values as constants for that property in this library.

Reply:
I understand the preference for US english spelling, but considering the british english spelling is ued for the property in the corresponding typescript interface, would parity with that not be more important?

Reply:
Aye, that sounds fine don't care about the spelling given that it matches TS. I just picked it because that link said it was correct for both US and British (not a preference for one or the other).